cereal/car.capnp

```diff
@@ -50,6 +50,7 @@ struct CarState {
   struct CruiseState {
     enabled @0: Bool;
     speed @1: Float32;
+    mainOn @2: Bool;
```

Comment:
I'm not sure this belongs here? What does this mean in general for cruise control systems?